### PR TITLE
Add PostgreSQL migration from AWS RDS & Aurora and Google Cloud SQL

### PR DESCRIPTION
**Pull request by runwuf**
Added tutorials of:
* Migrate from RDS PostgeSQL to Aiven PostgreSQL
* Migrate from Amazon Aurora PostgeSQL to Aiven PostgreSQL
* Migrate from Google Cloud SQL PostgeSQL to Aiven PostgreSQL

### CODE REVIEW

**Comment by ftisiot** [comment]
Hi @runwuf is this now ready to be reviewed? Thanks!

**Comment by lornajane** [comment]
@runwuf if this was anyone else's PR I'd close it on the basis of inactivity - but I really want to get this content published if we can. How can we help make this happen?

**Comment by runwuf** [comment]
@ftisiot @lornajane apology for lack of activity on this!  what I got stuck with is the avn cli status output when migrating from both aws rds/aurora and Google CloudSQL postgres, out of the box it WILL FAIL with a very vague status output by avn cli.  I've already found the root cause of one issue in the aiven-db-migrate tool and made a PR for it (but it needs to be fine tuned).  I haven't had a chance to fix that PR to get it merged and not to mention fix also need on avn cli.
What I am going to do is to suggest running aiven-db-migrate tool directly (where you need to be in the correctly configured network) instead of avn cli in these devportal guides.
I will update the PR tomorrow, expect to be ready to review next week in your time zone.  
Thank you for your patience.

**Comment by runwuf** [comment]
Apology for the long delay, this is now ready for review.  @ftisiot @TibsAtWork @lornajane

**Comment by lornajane** [comment]
Added the "help wanted" tag since I don't have the cloud accounts needed to run through these steps.

**Comment by ROConnor-Aiven** [comment]
@lornajane Sorry for the delay, finally got through this, it looks good but I don't seem to have permission to approve.

**Comment by lornajane** [comment]
Was it just that I didn't have your GitHub username to tag you as reviewer, can you approve now?

**Review by ROConnor-Aiven — APPROVED** [review]
Looks good, I was able to follow these steps.

**Comment by runwuf** [comment]
thanks @ROConnor-Aiven validated these steps.
@lornajane @ftisiot any feedback from devrel team on the wording, layout, screenshots and anything else?

**Comment by ftisiot** [comment]
I'm on it  now!

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @runwuf! 
Thanks for the contribution! 
I would like to open the discussion since you did a great job at dividing the repeated process in `aiven-db-migrate`. I would personally keep that as separate file and reference it via link from all the others (rather than incorporating). Since the set of steps to perform is always the same.

For all the others (e.g. `migrate-from-aws-aurora`) I would rephrase the start of the article to state what the reader is going to do in each case rather than a standard template on `aiven-db-migrate` itself. I would, for example specify what's is aurora with a link, that we're going to use aiven-db-migrate (with another link to the relevant devportal page) to perform it and that we are going to cover the Aurora setup to achieve the migration.

Summarizing, since the process is 90% the same, I would move from `migrate from aws aurora` to `prerequisite for aws aurora` and state only what's different in that setup? an example of what I'm saying is this https://developer.aiven.io/docs/products/kafka/kafka-connect/howto/gcs-sink-prereq.html

These are my thoughts, WDYT?

**Comment by runwuf** [comment]
Hi @ftisiot thank you for your feedback!
If I understand correctly, does this mean instead of the current structure, change it to let `migrate-aiven-db-migrate.rst` with prereq links to
- `aws-rds-prereq.rst`
- `aws-aurora-prereq.rst`
- `gcp-cloudsql-prereq.rst`

Let me know then I'll make the changes.

For running `aiven-db-migrate`, there might be additional cloud specific flags to skip unnecessary cloud specific roles and db to migrate, I have an open PR to the  aiven-db-migrate but it is not finalized yet.

**Comment by runwuf** [comment]
@ftisiot - I updated as per our discussion, please take a look again, thanks!
one of the check is failing on this file which I didn't touch, wondering if there's recent changes on sphinx/prose check?
```/home/runner/work/devportal/devportal/docs/products/mysql/howto/connect-from-cli.rst:61: WARNING: Duplicate explicit target name: "mysql documentation".```

**Comment by ftisiot** [comment]
Hi! I'll review it now.

Regarding the warning, we are aware of it, don't worry in this PR

**Comment by ftisiot** [comment]
Hi @runwuf ! I did a bit of reformatting to fix the text, I also moved the Python example in a standalone document because it can be used also without Aiven for PG. Could you please check the following:
* If all the info is still accurate
* If we can remove some of the files (the `migrate...aurora/rds/google cloud`)
* If the left menu makes sense
There's a link failure which shows the build as red. It's related to PG docs and I hope it will be fixed soon

**Review by ftisiot — APPROVED** [review]
LGTM!
